logger/src/commonMain/kotlin/com/wire/kalium/logger/KaliumLogger.kt

```diff
@@ -0,0 +1,109 @@
+package com.wire.kalium.logger
+
+import co.touchlab.kermit.Severity
+import co.touchlab.kermit.StaticConfig
+import co.touchlab.kermit.Logger as KermitLogger
+
+/**
+ * LoggerType in order from lowest to the highest severity level.
+ */
+enum class LoggerType {
+    /**
+     * A log level describing events showing step by step execution of your code that can be ignored during the standard operation,
+     * but may be useful during extended debugging sessions.
+     */
+    VERBOSE,
+
+    /**
+     * A log level used for events considered to be useful during software debugging when more granular information is needed.
+     */
+    DEBUG,
+
+    /**
+     * An event happened, the event is purely informative and can be ignored during normal operations.
+     */
+    INFO,
+
+    /**
+     * Unexpected behavior happened inside the application,
+     * but it is continuing its work and the key business features are operating as expected.
+     */
+    WARN,
+
+    /**
+     * One or more functionalities are not working, preventing some functionalities from working correctly.
+     */
+    ERROR,
+
+    /**
+     * Logger is Disabled.
+     */
+    DISABLED
+}
+
+class KaliumLogger(config: Config) {
+
+    private val kermitLogger: KermitLogger
+
+    init {
+        kermitLogger = KermitLogger(
+            config = StaticConfig(
+                minSeverity = config.severityLevel()
+            ),
+            tag = config.tag
+        )
+    }
+
+    val severity = config.severity
+
+    @Suppress("unused")
+    fun v(message: String, throwable: Throwable? = null) =
```

Comment:
Let's use the whole word instead of an abbreviation? Same thing goes for the rest of the methods.

```suggestion
    fun verbose(message: String, throwable: Throwable? = null) =
```

Reply:
I'm OK with it if that's the standard in the kotlin world :)

Reply:
So, thought on that but followed with the abbreviation due to all the logging/Loggers having it abbreviates (Log.i/Log.id) and also from the Kermit lib itself, but it would indeed make more sense having it as the whole word.

Will change it!